tests/rptest/tests/timequery_test.py

```diff
@@ -262,12 +277,17 @@ def set_up_cluster(self, cloud_storage: bool, batch_cache: bool):
             # Testing with batch cache disabled is important, because otherwise
             # we won't touch the path in skipping_consumer that applies
             # timestamp bounds
-            'disable_batch_cache': not batch_cache,
+            'disable_batch_cache':
+            not batch_cache,
 
             # Our time bounds on segment removal depend on the leadership
             # staying in one place.
-            'enable_leader_balancer': False,
-            'log_segment_size_min': 32 * 1024,
+            'enable_leader_balancer':
```

Comment:
nit: looks like these lines were accidentally wrapped

Reply:
It looks strange but YaPF did this, I verified by running it outside the editor.